pgjdbc/src/main/java/org/postgresql/core/v3/SimpleParameterList.java

```diff
@@ -43,12 +48,53 @@
   private static final byte TEXT = 0;
   private static final byte BINARY = 4;
 
-  SimpleParameterList(int paramCount, @Nullable TypeTransferModeRegistry transferModeRegistry) {
+  private static final ParameterContext NO_PARAMETER_CONTEXT = ParameterContext.EMPTY_CONTEXT;
```

Comment:
why create another constant reference to the existing constant?

Reply:
That is a left over. Fixed.